docs/DFAnumerics.rst

```diff
@@ -607,3 +610,69 @@ Finaly, the entrainment force reads:
 
 .. math::
     F_{k,i}^{\text{ent}} = -w_f\,(e_s+\,q_{k}^{\text{ent}}\,e_d)
+
+Adding forces
+--------------
+The different components are added following an operator splitting method.
+This means, the velocity of the particles are updated successively with the different forces.
+
+
+Adding artificial viscosity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If the viscosity option (``viscOption``) is set to 1, the artificial viscosity term is added first as described
+in :ref:`DFAnumerics:Artificial viscosity`. With ``viscOption`` set to 0, no viscosity is added. Finally, if
+``viscOption`` is set to 2, the artificial viscosity is added together with computing the SPH force (add link to description)
```

Comment:
```suggestion
If the viscosity option (``viscOption``) is set to 1, artificial viscosity is added first as described
in :ref:`DFAnumerics:Artificial viscosity`. With ``viscOption`` set to 0, no viscosity is added. Finally, if
``viscOption`` is set to 2, artificial viscosity is added during SPH force computation.  (TODO add link to description)
```
Please add the link

Reply:
why is there a first in the first sentence - it is added as described in ref:DFA..

Reply:
and maybe you could add that this is the default option?

Reply:
by first I mean it is the first thing we do in each time step